### ISSUE TITLE
CellDefinition's content type is wrong

### ISSUE THREAD

**Issue by andrico1234** [simonbengtsson/jsPDF-AutoTable#576]
I'm using jsPDF in my project and I'm creating a CellDefinition object like below

```
  const TableHeading: CellDefinition[] = [
    {
      content: "Title One",
      colSpan: 2
    },
    {
      content: "Title Two",
      colSpan: 2
    }
  ];
```

I'm getting a typescript error because `Type 'string' is not assignable to type 'Styles | undefined'.ts(2322)`

It looks like this is the issue:

```
interface CellDefinition {
    rowSpan?: number,
    colSpan?: number,
    styles?: Styles,
    content?: Styles,
}
```

I imagine that content shouldn't be of type `Styles`. I'll be happy to make a PR to update this if you're happy to confirm the type it should be. 

Thanks for making this package. It's exactly what I need for my project!

-A

**Comment by simonbengtsson** [comment]
It should not be a string? Rather a styles object?

**Comment by mmghv** [comment]
I'm sure it shouldn't be of type `Styles`, it's should be string but according to this line :
https://github.com/simonbengtsson/jsPDF-AutoTable/blob/e5bdf556dff97bd2c655d06c85d005276b0d7a38/src/models.ts#L157

It can be some type of an object with a `title` property, I'm not sure what object this line is accounting for though.

**Comment by simonbengtsson** [comment]
Oh, you mean the content? Didn't see that. Will fix it right away.

**Comment by simonbengtsson** [comment]
Fixed now in latest version 3.2.11. Thanks for reporting!

**Comment by andrico1234** [comment]
Thanks for the quick response!
